resources/titles.xml

```diff
@@ -6789,16 +6789,6 @@
         <title>A New Praetorian Laterculus from Rome</title>
       </bibl>
     </bibl>
-    <bibl type="v" xml:id="ZPE_145">
```

Comment:
I think you need to update your copy of the bibliography file before this can be merged; a number of things are missing from the copy you are trying to merge, and it seems to be attempting to remove them.